seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java

```diff
@@ -92,103 +92,33 @@ private JsonToRowConverter createNotNullConverter(SeaTunnelDataType<?> type) {
             case ROW:
                 return createRowConverter((SeaTunnelRowType) type);
             case NULL:
-                return new JsonToRowConverter() {
```

Comment:
The flink or spark serialization compiler does not support lambda, so this code cannot be updated